src/main/java/com/google/sps/image/ImageScorer.java

```diff
@@ -0,0 +1,106 @@
+package com.google.sps.image;
+
+import java.util.List;
+import java.util.Set;
+
+/** 
+ *  Custom class of scoring images using
+ *  the elements extracted using Cloud Vision API.
+ */
+public final class ImageScorer {
+
+  private static final float LANDMARK_SCORE = 0.8f;
+
+  private static final float LOGO_SCORE = 0.5f;
+
+  private static final float OCR_SCORE = 0.1f;
+
+  private static final float LABEL_SCORE = 0.2f;
+
+  private static final double EPS = 1e-5;
+
```

Comment:
The new line between these constants seems unusual to me. You can check this as an example maybe https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names
I didn't find any style guide that specifies the mandatory new line between variable declarations, but I can see that you declare all your class variables like this.

Reply:
In the Google Style Guide says this is optional. I took into account the readability, but I can create blocks of elements that refer to the same topic (label, landmark, ocr, logo scores). https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace